config/zones.json

```diff
@@ -644,8 +646,7 @@
       "coal": 9535,
       "gas": 30723,
       "geothermal": 0,
-      "hydro": 20281,
-      "hydro storage": 5645,
+      "hydro": 20053,
```

Comment:
I'm sorry to be picky, but why don't we breakdown spain storage capacity? It seems that Entsoe gives us live data for it.

Reply:
@brunolajoie Be picky, it's absolutely okay! :) I wasn't sure anyway whether I should ask you again to make things totally clear for Spain, because I still had some doubts myself :)

I had a look at the generation and capacity data on entso-e, and here is the reason: 

I would not split Spanish hydro, because the generation from "pumped storage" is reported within "hydro reservoir". There would be absolutely no generation from PS shown in the map, it would be included in the usual "hydro" category. Generation from PS is always empty on entso-e for Spain.

However, the consumption is reported for pumped strorage. You can see it in the negative area of the PS category at night. So if we had the capacity for "pumped storage" on the map, there will only be the negative consumption values, the positive side would never be filled, just like it is now.

I want to point at entso-e once again, even if that is not the source of the data for Spain now:

![image](https://user-images.githubusercontent.com/25743609/34944780-38c845c0-fa01-11e7-8621-45f5a8414fa5.png)

So according to entso-e, the generation capacity from **run-of-river** plus **hydro reservoir** is 20281 MW. This is also including the **generation** capacity of pumped storage. 
That is probably a result of using **some of the hydro reservoirs as pumped storages** as well, so there is some natural water intake throughout the entire year and a partial pumped storage water intake for a few hours during "pumped storage consumption". In the IRENA numbers, they might be included in the "mixed hydro" category.

Listing the pumped storages capacity seperately would create the impression that Spain has about 20 GW of hydro capacity and another 5.6 GW of pumped storage (or 3.28 pumped storage according to IRENA), that are already included in the ~20 GW.
That is why I assigned 0 to pumped storage, because only the consumption does appear in the graph. Generation is included in "hydro", therefore the "more correct" capacity is ~20 GW.

We only need the negative part of the pumped storage capacity according to the entso-e generation data. I think, this is probably not possible to realize at the moment?

Reply:
Super clear, thanks! Indeed, we havent programmed yet the ability to distinguish storage capacity from discharge capacity. Lets go with this!